lessons/codap-exploring/langs/en-us/pages/codap-exploration.adoc

```diff
@@ -1,50 +1,100 @@
 = CODAP Exploration
+++++
+<style>
+body.workbookpage .autonum { padding-top: 0.5rem; }
+body.workbookpage .fitb { padding-top: 0.7rem;}
+</style>
+++++
 
-Make sure you’re logged into the @starter-file{animals} in CODAP. The purpose of this exploration is to familiarize yourself with some of CODAP's features. Feel free to experiment and try things out!
+This page will help you familiarize yourself with some of CODAP's features. Check off each item once you have completed it. Feel free to experiment and try things out! Make sure you’re logged into the @starter-file{animals} in CODAP before beginning.
 
-@n See if you can make a graph. What did you select in order to do so?
+== Tables in CODAP
 
-@fitb{}{@ifsoln{I clicked on the "Graph" icon in the upper left-hand corner of my screen}}
+@n A table of data is shown here, with the title at the top of the table. What is the title? @fitb{}{@ifsoln{Animals-Dataset-1.5.1}}
 
-@fitb{}{}
+@n Move the table to a different location on the screen.
 
-@vspace{2ex}
+@n Minimize the table. (Hint: Try hovering your mouse over the table's title. What appears?)
 
-@n Move the table. Move the graph. How do you move things in CODAP?
+@n Now re-expand the minimized table.
 
-@fitb{}{@ifsoln{To move tables and graphs, I mouse over the top-most bar until my cursor turns into a hand. Then I can drag.}}
+@n Add a row to the table, then delete that same row.
 
-@fitb{}{}
+@n Move the `Age` column so that it is between `Fixed` and `Legs.`
 
-@vspace{2ex}
+@n Click on `Age` and, from the drop-down menu that appears, choose "Sort Ascending (A→Z, 0→9)." Now try "Sort Descending." 
 
-@n Minimize and maximize the table and the graph. How do you do that?
+@n How many animals have names that begin with S? @fitb{5em}{@ifsoln{6}}
 
-@fitb{}{@ifsoln{ To minimize, I select the `-` in the upper right-hand corner of the graph or table. To maximize, I reselect the `-`.}}
+@n Delete a column of the table.
 
-@fitb{}{}
+@n Close the table. __Can you get it back?__ (Hint: A drop-down menu appears when you click on "Tables" in the upper left.)
 
-@vspace{2ex}
+@n Create a new attribute. Is the column populated or empty? @fitb{15em}{@ifsoln{Empty.}}
 
-@n Drag an attribute (like `weight`, `name`, or `sex`) to a graph's axis - or use the drop-down menu that appears when you click on an axis. What happens?
+@n Name your new attribute. What did you choose? @fitb{15em}{}
 
-@fitb{}{@ifsoln{ _The randomly positioned points move to the confirguration that matches the data on the animals table._}}
+@n Do Undo (`Ctrl-Z` on PC, `Cmd-Z` on Mac) and Redo (`Ctrl-Y` on PC, `Cmd-Opt-Z` on Mac) work in CODAP? @fitb{}{@ifsoln{yess}}
 
-@fitb{}{}
+== Graphs in CODAP
 
```

Comment:
This whole section is pretty confusing to me... I started giving the following feedback but then a few bullets down realized you had other visions. Maybe we should discuss?

> See if you can make a graph OF WHAT? Let's be specific here, because students can only answer the questions that follow if they've chosen two variables, even though technically they can make a graph without selecting variables. It'll probably also be easier for teachers if all students make the same graph at this juncture.